README.md

```diff
@@ -8,6 +8,8 @@
 [![Test Coverage](https://codecov.io/github/Workiva/over_react/coverage.svg?branch=master)](https://codecov.io/github/Workiva/over_react?branch=master)
 
 > A library for building statically-typed React UI components using Dart.
+>
```

Comment:
I think it would help visibility-wise to have this be its own section, similar to the Dart 2 Migration guide:
```suggestion

---

```

<img width="803" alt="Screen Shot 2019-10-29 at 11 06 35 AM" src="https://user-images.githubusercontent.com/1713967/67795886-3c68e580-fa3c-11e9-90bf-f04d55119ba9.png">